src/pool/srv_target.c

```diff
@@ -472,13 +472,51 @@ pool_child_recreate(struct ds_pool_child *child)
 
 }
 
+struct ds_pool_flags {
```

Comment:
minor, not request change, "ds_pool_flags" looks generic name, ds_pool_flags_arg or sth maybe better

Reply:
OK, let me change the name.